charts/graphscope-store/templates/_helpers.tpl

```diff
@@ -86,6 +86,13 @@ Return the proper image name
      {{- $registryName = .global.imageRegistry -}}
     {{- end -}}
 {{- end -}}
+{{- if not $tag }}
+{{- if .imageRoot.DefaultTag }}
```

Comment:
It seems the new kv you added isn't used ("DefaultTag" .Chart.AppVersion)?
The `.imageRoot.DefaultTag` points to the `image: DefaultTag` in values.yaml?
So I guess if you install the charts without tag specified, you will have `latest` tag instead of `0.17.0` or `0.18.0` (the AppVersion)

Reply:
Maybe should change the `.imageRoot.DefaultTag` to `.DefaultTag`?